cmd/kubeadm/app/cmd/join.go

```diff
@@ -64,7 +64,7 @@ var (
 
 		* Certificate signing request was sent to apiserver and approval was received.
 		* The Kubelet was informed of the new secure connection details.
-		* Master label and taint were applied to the new node.
+		* Control Plane label and taint were applied to the new node.
```

Comment:
lets write this one like so:
`* Control plane (master) label...`

this is because the underlying label is "master" and it's not easy to change that at this point.

Reply:
I also think that it is reasonable.